rpcs3/Emu/SysCalls/lv2/sys_tty.cpp

```diff
@@ -22,7 +22,7 @@ s32 sys_tty_write(s32 ch, vm::ptr<const void> buf, u32 len, vm::ptr<u32> pwritel
 {
 	sys_tty.Log("sys_tty_write(ch=%d, buf_addr=0x%x, len=%d, preadlen_addr=0x%x)", ch, buf.addr(), len, pwritelen.addr());
 
-	if(ch > 15 || (s32)len <= 0) return CELL_EINVAL;
+	if(ch > 15 || len == 0) return CELL_EINVAL;
```

Comment:
It made me take a look into LV2. It seems that signed comparison is correct, but it doesn't return EINVAL, just CELL_OK if len is <= 0.